From 49b007b1f99a1a6aa83493380e6208f83e79d3bb Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 29 Dec 2016 12:56:59 +0800
Subject: [PATCH] MALI: midgard: RK: manual define configs

otherwise it won't defined since we don't include it in kconfig

Change-Id: I6886068f89d53f3ecf47846ed7435131fdd7960f
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 drivers/gpu/arm/midgard_for_linux/default_config.mk | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/gpu/arm/midgard_for_linux/default_config.mk b/drivers/gpu/arm/midgard_for_linux/default_config.mk
index c7469087f66d..52ed3ffdbc22 100644
--- a/drivers/gpu/arm/midgard_for_linux/default_config.mk
+++ b/drivers/gpu/arm/midgard_for_linux/default_config.mk
@@ -6,4 +6,14 @@ CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 export CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
+CONFIG_MALI_TIMELINE_DISABLED=y
 
+DEFINES = \
+	-DCONFIG_MALI_MIDGARD \
+	-DCONFIG_MALI_DEVFREQ \
+	-DCONFIG_MALI_EXPERT \
+	-DCONFIG_MALI_DEBUG_SHADER_SPLIT_FS \
+	-DCONFIG_MALI_DEBUG \
+	-DCONFIG_MALI_TIMELINE_DISABLED
+
+ccflags-y += $(DEFINES)
-- 
2.35.3

